### ISSUE TITLE
benchexec crashes with KeyError in benchexec/outputhandler.py, line 385

### ISSUE THREAD

**Issue by danieldietsch** [sosy-lab/benchexec#598]
When I run benchexec 3.3-dev (7a0ac2faa) on https://github.com/ultimate-pa/ultimate/blob/dev/releaseScripts/default/benchexec/student-concurrency.xml I get the following error:
```
 $ /storage/repos/benchexec-2/bin/benchexec --no-container --maxLogfileSize -1 --no-compress-results -N 14 ../benchexec/student-concurrency.xml
2020-09-07 17:00:16,960 - INFO - Unable to find pqos_wrapper, please install it for cache allocation and monitoring if your CPU supports Intel RDT (cf. https://gitlab.com/sosy-lab/software/pqos-wrapper).
2020-09-07 17:00:16,975 - WARNING - Turbo boost of CPU is enabled. Starting more than one benchmark in parallel affects the CPU frequency and thus makes the performance unreliable.

executing run set 'Automizer SVCOMP 2020'     (1093 files)
Traceback (most recent call last):
  File "/storage/repos/benchexec-2/bin/benchexec", line 27, in <module>
    sys.exit(benchexec.benchexec.main())
  File "/storage/repos/benchexec-2/bin/../benchexec/benchexec.py", line 470, in main
    sys.exit(benchexec.start(argv or sys.argv))
  File "/storage/repos/benchexec-2/bin/../benchexec/benchexec.py", line 73, in start
    rc = self.execute_benchmark(arg)
  File "/storage/repos/benchexec-2/bin/../benchexec/benchexec.py", line 348, in execute_benchmark
    result = self.executor.execute_benchmark(benchmark, output_handler)
  File "/storage/repos/benchexec-2/bin/../benchexec/localexecution.py", line 149, in execute_benchmark
    output_handler.output_before_run_set(runSet)
  File "/storage/repos/benchexec-2/bin/../benchexec/outputhandler.py", line 385, in output_before_run_set
    expected_result = str(run.expected_results[prop.filename])
KeyError: '../../../trunk/examples/svcomp/properties/unreach-call.prp'
```

``../../../trunk/examples/svcomp`` is a symlink to ``/storage/repos/svcomp/c`` from sosy-lab/sv-benchmarks@fbb4a6049b, so the property file is https://github.com/sosy-lab/sv-benchmarks/blob/fbb4a6049b44f2ed72716484aa92501bb06ec363/c/properties/unreach-call.prp.

**Comment by PhilippWendler** [comment]
Fixed yesterday by cf2ce2b, thanks for reporting!

**Comment by danieldietsch** [comment]
Thank you, now it works for me!
